.github/workflows/duplicate_pr.yml

```diff
@@ -0,0 +1,77 @@
+name: Duplicate PR
+
+#Duplication workflow must be run before PR branch is deleted
+on:
+  issue_comment:
+    types: [created, edited]
+
+jobs:
+  duplicate_pr:
+    name: Duplicate PR
+    if: ${{ github.event.issue.pull_request && startsWith(github.event.comment.body,'@dimagibot duplicate this PR') }}
```

Comment:
we can probably remove `@dimagibot ` and instead just use `duplicate this PR` as a comment to trigger this.

Reply:
[e3f7580](https://github.com/dimagi/commcare-core/pull/1259/commits/e3f7580b672d1da21361164a3d0a4d057c824fc7)